cmd/openshift-install/testdata/agent/image/manifests/default_manifests.txt

```diff
@@ -111,6 +111,7 @@ spec:
   clusterRef:
     name: ostest
     namespace: cluster0
+  cpuArchitecture: x86_64
```

Comment:
I think it would be useful to have a new integration test for the arm64 arch. There could be a new `arch` folder to store the new tests, one for each arch, to ensure that ISO image gets properly created, with the expected name

Reply:
@andfasano I added arm64_manifests.txt to the manifests folder, it seemed cleaner than creating subfolders. Happy to consider other options.

Reply:
@jeffdyoung that sounds reasonable to me, not sure if you still have to push it?